### ISSUE TITLE
ES Module errors

### ISSUE THREAD

**Issue by ashleigh-yeoman** [woofers/react-dialog-polyfill#25]
`react-dialog-polyfill` works perfectly fine in a Create React App application, but when moving to Vite, the following error occurs.
```
✘ [ERROR] No matching export in "node_modules/react-dialog-polyfill/lib/react-dialog-polyfill.module.js" for import "default"

    node_modules/react-dialog-polyfill/lib/index.module.js:3:9:
      3 │ import { default as Dropper } from './react-dialog-polyfill.module.js'
        ╵          ~~~~~~~
```
Looking at `node_modules/react-dialog-polyfill/lib/index.module.js`, I can see that `Dropper` is an unused import. While the simple solution would be to remove the import, I'm curious as to specifically _why_ this error is occuring. I can't find any similar instances of this particular issue elsewhere online, so I wonder if this is an issue with microbundle.

I have a reproduction repo [here](https://github.com/james-yeoman/react-dialog-polyfill-repro). If you've seen this problem before, I'd be interested in hearing why, for future reference in case another library has the same issue. If not, then removal of the unused import would fix this up and cross off one barrier to migrating from CRA to Vite.

**Comment by woofers** [comment]
@james-yeoman Thanks for the report and the minimal reproduction.  This `Dropper` export appears to be a leftover from when I moved away from Webpack, and is a typo. 

Essentially the package is trying to import something that does not exist.  The reason it likely fails in Vite while working in create-react-app is that Vite will use the ES Module version (`index.module.js` with this typo) while I think CRA will use the UMD bundle (`index.js`) without this typo.

I have pushed `v1.1.6` of `react-dialog-polyfill`  which should hopefully resolve this.  Let me know if you still encounter any issues.
